src/hooks/useIsRewardClaimed.ts

```diff
@@ -0,0 +1,12 @@
+import { RolePlatform } from "../types"
+import { useGetUserRewards } from "./useGetUserRewards"
+
+export const useIsRewardClaimed = (platformId: RolePlatform["id"]) => {
```

Comment:
`platformId` is a little bit misleading here in my opinion, since `platformId` is the ID of a specific platform type (e.g. Discord or Telegram), and not a reward's ID